Documentation/network/servicemesh/ingress.rst

```diff
@@ -19,6 +19,24 @@ routing and for TLS termination.
     The ingress controller creates a service of LoadBalancer type, so
     your environment will need to support this.
 
+Cilium allows you to specify load balancer mode for the Ingress resource:
+
+- ``dedicated``: The Ingress controller will create a dedicated loadbalancer
+  for the ingress.
+- ``shared``: The Ingress controller will use a shared loadbalancer for all
+  Ingress resources.
+
+Each load balancer mode has its own benefits and drawbacks. The shared mode saves
+resources by sharing a single LoadBalancer config across all Ingress resources in
+the cluster, while the dedicated mode can help to avoid blast radius of a single
+Ingress resource affecting others.
```

Comment:
What is "blast radius"?

I know what the words mean and the general sense, but it feels a bit vague. What kind of problem is it protecting against, is it user error? Cilium bugs? Something else?

Reply:
^^ In general, wording like "blast radius" is convenient in casual discussions, but in documentation it gives an impression to the reader that the actual behaviour is unknown. However, it is the job of the documentation to describe exactly what the behaviour is, so ideally this should be factual and based on statements of expected behaviour. This can help to assure the reader that the code will work correctly or warn them specifically about which conditions they can guard against by using dedicated mode.